en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -74,12 +74,10 @@ If using a Spektrum satellite receiver, this should be connected to the flight c
 
 ## Power
 
-The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries. 
 Connect the 6pin connector of the *HW\_PM* module to the flight control `Power1` interface.
 
-> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_plus.md#issue_pm_unfused)).
-   - Power **must** be turned off while connecting peripherals.
-   - Improper wiring can lead to *personal harm* or equipment damage!
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_plus.md#issue_pm_unfused)). Power **must** be turned off while connecting peripherals.
```

Comment:
@cuhome as for nano, not clear (to me) about the solution to this part.

Reply:
This is the wrong description, it uses a restrictive valve, and the usb uses tvs diodes; it is safe, it has a fuse. unfuse is DSM / SBUS / RSSI interface pin1.